lib/r13n.dart

```diff
@@ -1,3 +1 @@
-library r13n;
```

Comment:
Instead of removing it we can keep it in and just add a library comment? This will show nicely in generated dart docs as well.